trans/templates/home.html

```diff
@@ -37,7 +37,11 @@ <h2>{{ contest.title }}</h2>
                 </thead>                
                 <tbody>
                 {% for task in contest.tasks %}
-                    <tr>
+                  <tr
+		    class="task-row"

```

Comment:
Nit: indentation looks off? (Mix between tabs and spaces in this file)

Reply:
Done.